avatar/templatetags/avatar_tags.py

```diff
@@ -94,7 +94,7 @@ def primary_avatar(user, width=settings.AVATAR_DEFAULT_SIZE, height=None):
     else:
         kwargs["height"] = height
 
-    url = reverse("avatar_render_primary", kwargs=kwargs)
+    url = reverse("avatar:render_primary", kwargs=kwargs)
```

Comment:
changing the reverse urls probably means that we will need to release it as part of a new major version number.

Reply:
Yes, but now i need to solve tests :D

Reply:
Sure. If we make a new major release, we probably need to add other things as well. Or we just keep the PR open for now if there is nothing else we want to change/fix right now.